myhpom/models/document.py

```diff
@@ -149,3 +150,14 @@ def authorized_client_ip(self, ip_address):
                 if ip_address in ip_range:
                     return True
         return False
+
+
+def abort_document_runs_on_delete(sender, instance, using, **kwargs):
+    """Any document runs associated with a DocumentUrl should be aborted at CF on delete."""
+    if instance.cloudfactorydocumentrun_set.exists:
```

Comment:
This should be `instance.cloudfactorydocumentrun_set.exists()`? Add a test to the model tests that verifies that either that `tasks.CloudFactoryAbourtDocumentRun.delay` is called (mock-like), or that it happened (requests-mock like).

Reply:
re/exists: oops, yeah.

Reply:
The model tests don't call the tasks at all, so I'm not sure what you're looking for.

Reply:
Oh, you mean for the DocumentUrl model, not for the CFDocumentRun model. Ok, that makes sense.